client/js/components/truncate-text.js

```diff
@@ -34,25 +37,29 @@ const truncateText = (caption) => {
 
     caption.classList.add(truncateClass);
 
-    if (hasBeenEllipsified(caption)) {
-      caption.parentNode.insertBefore(truncateControl, caption.nextSibling);
-      const truncated$ = fromEvent(truncateControl, 'click').fold((isClosed) => !isClosed, false);
+    hasBeenEllipsified(caption).then((value) => {
+      if (value) {
```

Comment:
hasBeenEllipsified returns false if the text isn't truncated. In which case, we don't need the control